README.md

```diff
@@ -1 +1,3 @@
-# private_repo
\ No newline at end of file
+# private_repo
+
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[heading-increment] Heading levels should increment by one level at a time](https://app.codacy.com/gh/codacy-acme/private_repo/pullRequest?prid=5357195)